kyaml/fn/framework/command/command_test.go

```diff
@@ -99,6 +105,7 @@ metadata:
 func TestCommand_standalone_stdin(t *testing.T) {
 	var config struct {
 		A string `json:"a" yaml:"a"`
+		B int    `json:"b" yaml:"b"`
```

Comment:
Including this test with a different type because nonstring types in test kustomize fn command are causing PR tests to fail in `cmd/config/internal/commands/e2e/e2e_test.go`. The failure is at [yaml.Unmarshal`in `kyaml/fn/framework/processor.go](https://github.com/kubernetes-sigs/kustomize/blob/master/kyaml/fn/framework/processors.go#L138) `.
I'm not really sure what change in my PR could have caused this e2e test to fail when it isn't in master, but this test also fails when run against master. So the tag is getting added within the kyaml package but it doesn't seem like that's due to any of my changes.
I'm wondering nonstring types aren't expected in the function config.

```
=== RUN   TestRunE2e/exec_function_args
    testutil.go:55: 
        	Error Trace:	testutil.go:55
        	            				e2e_test.go:737
        	Error:      	Received unexpected error:
        	            	exit status 1
        	Test:       	TestRunE2e/exec_function_args
        	Messages:   	[yaml: unmarshal errors:
        	            	  line 6: cannot unmarshal !!str `1` into int
        	            	  line 6: cannot unmarshal !!str `true` into boolError: exit status 1
        	            	]
```